services/paymaster/src/bin/config_sync.rs

```diff
@@ -71,7 +82,46 @@ async fn insert_app(user_id: &Uuid, name: &str) -> Result<Uuid, sqlx::Error> {
     Ok(app.0)
 }
 
-async fn insert_domain_config(app_id: &Uuid, domain: &Domain) -> Result<Uuid, sqlx::Error> {
+async fn insert_or_update_domain_config(
+    app_id: &Uuid,
+    domain: &Domain,
+) -> Result<Uuid, sqlx::Error> {
+    let existing_domain_config = sqlx::query_as::<_, (Uuid,)>(
+        "SELECT id FROM domain_config WHERE app_id = $1 AND domain = $2",
+    )
+    .bind(app_id)
+    .bind(domain.domain.to_string())
+    .fetch_optional(pool())
+    .await?;
+
+    if let Some(domain_config_result) = existing_domain_config {
+        let id = domain_config_result.0;
+        let res = sqlx::query(
```

Comment:
i think we should run `cargo sqlx prepare` whenever you update queries so that sql metadata gets updated and queries get validated. this prevents any schema errors in the queries at runtime

Reply:
we should use `query!` and/or `query_as!` to get the compile-time checking